### ISSUE TITLE
Access Div control by id

### ISSUE THREAD

**Issue by mr-rajeshkumar** [Microsoft/WinAppDriver#651]
Hi 
I am trying to find DIv control have id like below:
<div id="activationViewTitle" class="test" data-bind="text: Title"><RunTime change Title></div> 

Code using to access this <div> control: session.FindElementByAccessibilityId("activationViewTitle").Text 

But session not able to find this control. When I try to read PageSource, it showing element as 

<Text AcceleratorKey="" AccessKey="" AutomationId=""            ClassName="" FrameworkId="MicrosoftEdge" HasKeyboardFocus="False"

            HelpText="" IsContentElement="True" IsControlElement="True" IsEnabled="True"

            IsKeyboardFocusable="False" IsOffscreen="True" IsPassword="False" IsRequiredForForm="False" ItemStatus="" ItemType="" LocalizedControlType="text"

            Name="Activate POS" Orientation="None" ProcessId="9240" RuntimeId="42.132446.2.3" x="391" y="361" width="322" height="32" />

Strange, this conversion completely ignore the ID and that's why failing the FindElementByAccessibilityId.

Any Idea how to fix this issue.
